lahja/endpoint.py

```diff
@@ -368,13 +554,14 @@ def broadcast(self, item: BaseEvent, config: Optional[BroadcastConfig] = None) -
             # Internal events simply bypass going through the central event bus
             # and are directly put into the local receiving queue instead.
             self._internal_queue.put_nowait((item, config))
-        else:
-            # Broadcast to every connected Endpoint that is allowed to receive the event
-            compressed_item = self._compress_event(item)
-            for name, connector in self._connected_endpoints.items():
-                allowed = (config is None) or config.allowed_to_receive(name)
-                if allowed:
-                    connector.put_nowait((compressed_item, config))
+            return
+
+        # Broadcast to every connected Endpoint that is allowed to receive the event
+        compressed_item = self._compress_event(item)
+        for name, remote in list(self._outbound_connections.items()):
```

Comment:
*nitpick*.  In another place you use `copy`.  Here you cast to a `list`.  Can we use a common mechanism for this (like maybe `tuple()` because I love them so much more than lists :smile: )